app/views/v1/subdomain_util.py

```diff
@@ -0,0 +1,55 @@
+from typing import Set
+
+import json
+
+import flask_jwt_extended
+from flask import request, url_for
+
+import app.utils.authentication_utils as authentication_utils
+import app.actions as actions
+import app.db_models as db_models
+
+from . import bp
+from app.utils.ct_search import get_subdomains_from_ct
+from app.actions.add_targets import add_targets
+
+
+@bp.route(
+    "/api_add_subdomains/<int:target_id>",
+    methods=["POST", "DELETE"]
+)
+@flask_jwt_extended.jwt_required
+def api_add_subdomains(target_id: int):
```

Comment:
Add secondary endpoint. This endpoint does this once, the other endpoint should set this to be a period action.